doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -209,70 +209,99 @@ infrastructure itself from the workloads compute resources.
 <a name="3.2.2"></a>
 ### 3.2.2 Container Networking Services
 
-As all production networking solutions for Kubernetes are based on CNI plugins
-and the implicit requirement in [4.2.2 Virtual Network Interface
-Specifications](../../../ref_model/chapters/chapter04.md#422-virtual-network-interface-specifications)
-that documents the requirement to have the capability to attach several network
-interfaces to the pods, the CNTT architecture must support a CNI metaplugin/CNI
-multiplexer.
+Kubernetes networking is considered an "extension" to the core functionality,
+and is managed through the use of plugins [Network
+Plugins](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/).
+For hardware resources that are needed by Kubernetes applications, [Device
+Plugins](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/)
+can be used to manage those resources and advertise them to the kubelet for use
+by the Kubernetes applications. This allows resources such as "GPUs,
+high-performance NICs, FPGAs, InfiniBand adapters, and other similar computing
+resources that may require vendor specific initialization and setup" to be
+managed and consumed via standard interfaces.
+
+Figure 3-2 below shows the main building blocks of a Kubernetes networking solution:
+- **Kubernetes Control Plane**: this is the core of a Kubernetes cluster - the
+apiserver, etcd cluster, kube-scheduler and the various controller-managers. The
+control plane (in particular the apiserver) provide a centralised point by which
+the networking solution is managed using an abstract management API.
```

Comment:
I do not see an abstract API in Kubernetes to manage networking solutions.

Reply:
ok maybe we are using the term differently then - for me, the Kubernetes APIs, with their extensibility through CRDs or AA are an abstract API to manage the networking solutions within Kubernetes (and beyond - e.g. for external load balancers).  What do you see an abstract API "doing"?

Reply:
For me an abstract API is independent from its implementation. I think it is the subject of the API operations where our misunderstanding is. We should have 3 API-s: 

1. An API to manage the LCM of different Kubernetes extensions including ones related to networking. 
2. An API to administer network related configurations. Tibycally for things done my network engineering, like IP range allocations.
3. An API to consume the networks. 
In Kubernetes 2) and 3) are collapsed into one API what enables the usage of implementation specific annotations, this is not an abstract API.

Reply:
Let's consolidate this conversation here: https://github.com/cntt-n/CNTT/pull/2121#discussion_r552078730

Reply:
@CsatariGergely removed the term "abstracted" from this section in https://github.com/cntt-n/CNTT/commit/f16f1ca30506e50c2840cb7f4ed797982f93aaf5.  OK to resolve?